middlewares/authorizeBot.js

```diff
@@ -1,7 +1,21 @@
 const botVerifcation = require("../services/botVerificationService");
-const { CLOUDFLARE_WORKER } = require("../constants/bot");
+const { CLOUDFLARE_WORKER, CRON_JOB_HANDLER } = require("../constants/bot");
 
-module.exports = async (req, res, next) => {
+const verifyCronJob = async (req, res, next) => {
```

Comment:
this function does not check the time of the request. This means that the function could be called multiple times, even if the cron job is only supposed to run once.

Reply:
Thanks for the reminder we still have to decide on the time for running the corn job. Will Track it over the following issue: https://github.com/Real-Dev-Squad/website-backend/issues/1290